mkosi/mkosi.conf.d/debian-ubuntu/mkosi.conf.d/knot.conf

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+# New package needed for TEST-75-RESOLVED
+
+[TriggerMatch]
+Distribution=debian
+Release=!bullseye
+Release=!bookworm
+Release=!trixie
+# TODO: rename to oldstable once Forky ships
+Release=!stable
```

Comment:
So, maybe better to use codename in the main file rather than `testing`, `stable`, or `oldstable` ?

Reply:
But it is OK to do that later, if you do not want to do now.

Reply:
In the mkosi.yml it's a bit more convenient to use the aliases, so that there's no need to change them, but it's not a a big deal either way